eucaconsole.init

```diff
@@ -46,10 +46,21 @@ PROCESS=eucaconsole
 LOGFILE=/var/log/eucaconsole_startup.log
 PIDFILE=/var/run/eucaconsole/eucaconsole.pid
 NGINXPIDFILE=$(grep nginx.pid /etc/eucaconsole/nginx.conf | awk {'print $2'} | sed s'/\;//')
+MEMCACHEDPIDFILE=/var/run/memcached/eucaconsole-memcached.pid
```

Comment:
This is a separate copy of memcached, so put it in /var/run/eucaconsole instead.